startup/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+#from appprofile.models import Profile
+# Create your models here.
+class startup(models.Model):
```

Comment:
`Startup` not startup @naveennvrgup

Reply:
these are the commits of @anshsrtv. i hav just cleaned his old PR #33

Reply:
will fix it

Reply:
sir this is cleanup PR of PR #33 made by @anshsrtv. for some reason he have deleted these files on commit d8f9e95(if u check PR #33) for which u have been asking changes for. So these files dont exit in this branch. I just reset --soft to the commit he added the outdated frontend files and removed them and made a PR.